### PR TITLE
Harrison Qi add ReadME translation (Chinese)

### PR DESCRIPTION
**Pull request by HarrisonQi**
# Description

Add Chinese translation. Make it easier for Chinese game developers to learn and use the Flame framework.

Also provides links to switch between multiple languages.

## Type of change

add Chinese translation(README-ZH.md)

### CODE REVIEW

**Comment by spydon** [comment]
This will be very hard to keep up to date. Or we will have to ping you as soon as we update something so that you can do the corresponding update.

File: README-ZH.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+一款极简的Flutter游戏引擎.
+
+感激你对我们的任何支持! 评论, 建议, 提issues, Pull Request(贡献&推送)! 给我们点一颗Star, 支持我们! 
+
+## 问题互助
+
+我们在Fireslime的Discord上有一个互助频道, [点击加入](https://discord.gg/pxrBmy4). 现在我们也有了[FAQ](FAQ.md), 所以提问前请先在这里看看你有没有需要的答案.
+
+国内用户欢迎加入[Flame QQ交流群](https://jq.qq.com/?_wv=1027&k=5ETLFm3)(用户不多, 欲入从速)
```
Comment (spydon) [inline README-ZH.md:17]: Do we want unofficial chat links here?

Comment (HarrisonQi) [reply README-ZH.md:17]: There are very few discord users in China. The main chat rooms in China are WeChat and QQ. I hope to provide a place for Chinese users to communicate. If you mind, I will delete it immediately😃😃

**Comment by HarrisonQi** [comment]
> This will be very hard to keep up to date. Or we will have to ping you as soon as we update something so that you can do the corresponding update.

I noticed earlier that your ReadMe is updated about 2 or 3 times a month.
Normally, I will respond as soon as possible after receiving the notification. This year I will update the Readme at least once a month.
If there is a good ping method (such as email), I think there is no problem.

**Comment by erickzanardo** [comment]
I find a great idea to have translations from our documentations, but like @spydon , I am also worry that we would no be able to keep up with updating it, specially now that we are working towards a stable version of flame, and will update a lot of documentation.

Also, if the objective of this is make it easier for Chinese folks to learn Flame, I think it would be better to translate the files under `doc`, as here on there README there is very minimal information on how Flame and its features works.

@impulse if not much to ask, can you please help reviewing us this?

File: README-ZH.md (hunk 1 above)
Comment (erickzanardo) [reply README-ZH.md:17]: I am ok with this been here, as long as we state that this is not an official communication group.

**Comment by erickzanardo** [comment]
>I find a great idea to have translations from our documentations, but like @spydon , I am also worry that we would no be able to keep up with updating it, specially now that we are working towards a stable version of flame, and will update a lot of documentation.

Or we could simply add the following text on the start of the translated file:

```
This file is translated by the community and it is valid for Flame version 0.21.0
```

Or something like that

**Comment by impulse** [comment]
@erickzanardo sounds fine as long as it's explicitly stated its community maintained and last updated for Flame version X. As for the unofficial chat links I am pretty sure you cannot access Discord without VPN in mainland China.

File: README-ZH.md (hunk 1 above)
Comment (HarrisonQi) [reply README-ZH.md:17]: Ok, I'll declare that this is an unofficial organization

**Comment by HarrisonQi** [comment]
I think the above point is right👍

I just did two things:
1. Added supported version numbers.
2. Community translation statement.

You can check this version([`a7f3a77`](https://github.com/flame-engine/flame/pull/350/commits/a7f3a77e91b07270b2b668ba00ffc820b1400347))😃😃😃

**Comment by impulse** [comment]
lgtm 👍

**Comment by HarrisonQi** [comment]
> lgtm 👍

Thanks😄😄😄

**Comment by erickzanardo** [comment]
Oh, there is just one thing missing, you need to base your PR to our develop branch

**Comment by HarrisonQi** [comment]
Fix PR branch [here](https://github.com/flame-engine/flame/pull/354)🖖

**Comment by erickzanardo** [comment]
@HarrisonQi you don't need to close this PR and open a new one to change the base of your PR, you just need to edit your PR here on the Github and select our develop branch instead of the master.

I would rather merge this PR instead of the one you opened, as we have all the discussions on this PR.

**Comment by HarrisonQi** [comment]
@erickzanardo Ok, thank you🙂

**Pull request merged**